.github/PULL_REQUEST_TEMPLATE.MD

```diff
@@ -18,6 +18,6 @@ Thank you for your contribution to Braintree.
  > If you're an engineer for PayPal or Braintree, complete this section. For PRs from the community, please ignore!
 
 **When do these changes need to be released?** 
-[ ] ASAP (let's discuss outside this PR)
-[ ] Soon, here's our target release date: <DD/MM/YYYY>
-[ ] Whenever, no rush
+ - [ ] ASAP (let's discuss outside this PR)
```

Comment:
Nit - I think we generally _don't_ want to move PR discussion outside of a PR (like Slack or email if we can help it)

Reply:
Also would argue we don't need this section in the PR template, but will defer to the group

Reply:
What about this instead?

```
Ticket Priority

P4
P3
P2
P1
P0
```

Reply:
This was added by @hollabaq86 in response to some urgent PR/release requests. I am fine to remove it or leave it 🤷‍♀️ - the P0-P4 is a little too internal in my opinion, since this template is used for external contributors as well

Reply:
Do we know what the original motivation was to add ?

> When do these changes need to be released?